### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by krithigasr**
Updated readme file with the latest information.

### CODE REVIEW

**Review by HongBo713 — APPROVED** [review]

**Review by aditibhardwaj98 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,15 +1,111 @@
...
+## Application Overview
 
-***********************************************************
-Authors:
-1. Mia Wei (GitHub: MiaYWei)
-2. Krithiga Subramaniam Ravichandran (GitHub: krithigasr)
-3. Hongbo Pang (GitHub: HongBo713)
-4. Aditi Bhardwaj (GitHub: aditibhardwaj98)
+UNO Express is a fun and interesting game, traditionally played by a group of 2 to 4 people with a  deck of printed cards. We are proposing to develop this interactive game in C programming  language which supports a single player competing against a computer opponent. Playing games online  with computers could speed up our responses and develop logic skills. 
+
```
Comment (MiaYWei) [inline README.md:13]: Seems like there are some double character space between two words. Let's check it again, and remove them.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,111 @@
...
+And during the COVID-19  pandemic, we can use this game to keep ourselves entertained while maintaining our social circle.  What’s more, this game will keep you engaged and help reduce stress. This time you neither need to  fret over saying UNO before you drop your penultimate card nor wait until the deck is shuffled.
+
+By  this project, we intend to develop a UNO game engine that can make choices xadhering  to the rules of the game. 
+This game is similar to the well known UNO game but following modifications are made to deliver this game efficiently as a single player game with a computer opponent.
+
+* UNO Express card game has 56 cards, we removed the 'Reverse' action card as it doesn't make much of a sense in a two player game.So,
```
Comment (MiaYWei) [inline README.md]: 1. "make much of a sense"? or "make much sense"
2. "two player" or "two-player"

File: README.md [hunk 3]
```diff
@@ -1,15 +1,111 @@
...
+  the total card count is 52.
+* Human player always starts the game first.
+* When the Draw two action card is played, the next player automatically loses his turn, gets two cards added to his current set
```
Comment (MiaYWei) [inline README.md]: "Draw two" - We don't have "Draw Two", only have "Draw One" and "Wild Draw Two".

File: README.md [hunk 4]
```diff
@@ -1,15 +1,111 @@
...
+
+By  this project, we intend to develop a UNO game engine that can make choices xadhering  to the rules of the game. 
+This game is similar to the well known UNO game but following modifications are made to deliver this game efficiently as a single player game with a computer opponent.
+
+* UNO Express card game has 56 cards, we removed the 'Reverse' action card as it doesn't make much of a sense in a two player game.So,
+  the total card count is 52.
+* Human player always starts the game first.
+* When the Draw two action card is played, the next player automatically loses his turn, gets two cards added to his current set 
+  set of cards and the current player can choose the color to continue the game (same as a normal UNO game).
+* When the draw one action card is played, the next player by default loses his turn and a card will be added to his set of cards.
```
Comment (MiaYWei) [inline README.md]: Let's capitalize the first character of each word for the action card: "draw one"

File: README.md [hunk 5]
```diff
@@ -1,15 +1,111 @@
...
+7. These are the keyboard inputs with which human can play the game
+   * To request a card from the Draw pile, Press N or n.
+   * To end turn, Press E or e.
+   * To discard a normal card - Say you want to discard(RED,3), Press R3.
+   * To discard action cards, 
+     * Skip cards- Combine the first letter of the color with 'S', so if you want to discard(RED,SKIP), Press RS.
+     * Draw one cards- Combine the first letter of the color with 'O', so if you want to discard(RED,DRAW-ONE), Press R0.
+     * Wild cards- Press AW.
+     * Wild cards- Press AT.
+     (A above stands for Action cards, W stands for Wild, T stands for draw Two)
```
Comment (MiaYWei) [inline README.md:67]: Wild and Wild Draw Two

image

File: README.md [hunk 6]
```diff
@@ -1,15 +1,111 @@
...
+
+## Installation
+
+Complete instructions regarding installation of this project can be found here [UNO Express Card Game User Documentation](https://github.com/MiaYWei/Group_G_UNO-Card-Game/wiki/User-Manual).
+
+### Compile and Build Instructions
+
+The following make commands are available to help compile and build the application:
+
+* `make all` (default) Will build the application and test application.
```
Comment (MiaYWei) [inline README.md:93]: Let's lowercase of word "Will"

File: README.md [hunk 7]
```diff
@@ -1,15 +1,111 @@
...
+
+To run the application after installation, execute the application in the `bin` directory with  the command `UNO-Card-Game.exe`.
+
+
+Additional details can be found in the [user and developer documentation](https://github.com/MiaYWei/Group_G_UNO-Card-Game/wiki).
+
+### Test Application
+
+To run the test files, execute the application in the `bin` directory with `UNO-Card-Game_Test`. All test results are logged to the file 'test_log' in the same directory.
+
```
Comment (MiaYWei) [inline README.md:105]: "All test results are logged to the file 'test_log' in the same directory." - just currently. I think we need to relocate it to test-data folder. I will make the change today.

File: README.md [hunk 8]
```diff
@@ -1,15 +1,111 @@
...
+* doc - Includes the designed flowhcart for the application.  The main documentation can be found on the [UNO Express Card Game Documentation](https://github.com/MiaYWei/Group_G_UNO-Card-Game/wiki/User-Manual).
+* include - The main header files are listed below:
+  *  All Header files are included here that are used throughout the application.
+* src - Contains the definition for all functions.
+* test
+  * include - Contains the declarations for all unit tests.
+    * Includes the header files for the test files.       
+  * src - Contains the definitions for all unit tests.  
+    * test.c - Main file that calls the test functions in other files.
```
Comment (MiaYWei) [inline README.md]: Can we make it similar to:
image

**Review by MiaYWei — APPROVED** [review]
Left some suggested comments.

File: README.md (hunk 7 above)
Comment (krithigasr) [reply README.md:105]: Yes, we can make the change later when you're done with the modifications

File: README.md (hunk 8 above)
Comment (krithigasr) [reply README.md]: Do you want to explain all the folders?

File: README.md (hunk 6 above)
Comment (krithigasr) [reply README.md:93]: done

File: README.md (hunk 3 above)
Comment (krithigasr) [reply README.md]: I missed  adding Wild to it, normally its called as Draw two . I'll make the change for clarity

File: README.md (hunk 2 above)
Comment (krithigasr) [reply README.md]: Both are right!

File: README.md (hunk 1 above)
Comment (krithigasr) [reply README.md:13]: Made the changes, it shouldn't be a problem when it's rendered anyway.

**Pull request merged**